core/frontend/package.json

```diff
@@ -89,6 +91,7 @@
     "@itwin/object-storage-core": "^2.3.0",
     "@itwin/core-i18n": "workspace:*",
     "@itwin/webgl-compatibility": "workspace:*",
+    "@itwin/ecschema-rpcinterface-common": "workspace:*",
```

Comment:
can this really be a direct dep?

Reply:
It's use case is to register a default schema rpc locater for iModelConnections. It's internal-facing to me. If there's an application that wants to consume a different interface version for locating schemas, hence having a different dep of this pkg, they could retrieve the schemaContext and register this different locater.

Reply:
doesn't this intrinsically depend on ECSchemaRpcInterface which is static? and thereforce would cause us to make this a peer?

Reply:
I've switched it over to a peerDep